docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/stepper/accessibility/
+---
+
+## Accessibility
+
+The Stepper component has been designed with accessibility in mind. It can be used with keyboard navigation and includes properties that enhance the experience for users of assistive technologies.
+
+The following props provide additional information to screen readers, enhancing the accessibility of the respective component. By using them, you can ensure that users who rely on assistive technologies receive the necessary context and information about the component's purpose and functionality.
+
+- The `ariaLabel` prop allows you to specify an `aria-label` attribute of the component.
```

Comment:
Maybe here we should say that the `aria-label` is for the "value" and not the whole component? Wouldn't that make more sense? 🤔